editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -90,6 +100,7 @@ export interface PreviewStore extends Store.Store {
   saveToEmail?: string;
   overrideAnswer: (fn: string) => void;
   requestedFiles: () => FileList;
+  autoAnswerableInputs: (id: NodeId) => Value | undefined;
```

Comment:
`autoAnswerableOptions` and `autoAnswerableFlag` output a node ID which serves as the "answer ID" we can record in the breadcrumb of the auto-answered node, but in the case of input component types we want the breadcrumb to capture the actual user-entered value. 

The `Value` type works well as a bit of a "catch all" here, alternatively we could do a union based on the specific supported component type models like `Address | Contact | string | number` ? Thoughts/preferences?